<?xml version="1.0" ?>

<launch>
    <arg name="veh" default="husky1" />

    <node ns="$(arg veh)" pkg="subt_rl" type="denoise_mmwave.py" name="denoise_mmwave" output="screen">
        <param name="method" value="cgan" />
        <param name="model" value="0827_1851.pth" />

        <remap from="mm_laser" to="RL/scan_mmwave" />
        <remap from="recon_laser" to="RL/denoised_mmwave" />
    </node>


</launch>